src/test/java/org/mule/extension/file/integration/FileListTestCase.java

```diff
@@ -77,13 +79,15 @@ public void listRecursiveWithSizeCheckWithFileStillBeingWritten() throws Excepti
 
   @Test
   public void listWithDirectoryWithoutReadPermission() throws Exception {
+    assumeFalse(IS_OS_WINDOWS);
     temporaryFolder.newFolder("forbiddenDirectory").setReadable(false);
     List<Message> messages = doListWithSizeCheck(".", true);
     assertRecursiveTreeNode(messages);
   }
 
   @Test
   public void listWithFileWithoutReadPermission() throws Exception {
+    assumeFalse(IS_OS_WINDOWS);
```

Comment:
This one is not trying to make a directory unreadable, only a file. From what I have read for the explanation of the other PR linked, the problem was with directories. Maybe there is an actual problem with this test

Reply:
This happens in directories and files.